.gitignore

```diff
@@ -1 +1 @@
-hieradata/nodes/example-puppet-master.yaml
```

Comment:
is the removal of this line intentional?

Reply:
@fvoges Yes. That file doesn't exist anymore and was a relic from the old version of the control-repo from two years ago.